build/env/install-grpc.sh

```diff
@@ -57,6 +57,7 @@ if ! [ -x "$(command -v grpc_cpp_plugin)" ]; then
     ensure_git
     git clone --recurse-submodules -b v1.28.1 https://github.com/grpc/grpc
     cd grpc
+    make plugins -j 12
```

Comment:
12 core is quite an optimistic assumption, let's just use however many there are: `$(nproc)`
```suggestion
    make plugins -j $(nproc)
```